config.json

```diff
@@ -70,6 +70,21 @@
         "transforming"
       ]
     },
+    {
+      "uuid": "7961c852-c87a-44b0-b152-efea3ac8555c",
+      "slug": "isbn-verifier",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
+        "type_conversion",
+        "conditionals",
+        "strings",
+        "arrays",
```

Comment:
Not sure that `arrays` is a relevant topic for this one (might be for certain solution implementations, but not integral to the problem definition).